numpy/_core/src/multiarray/nditer_pywrap.c

```diff
@@ -576,66 +582,61 @@ npyiter_convert_op_axes(PyObject *op_axes_in, int nop,
     return 1;
 }
 
-/*
- * Converts the operand array and op_flags array into the form
- * NpyIter_AdvancedNew needs.  Sets nop, and on success, each
- * op[i] owns a reference to an array object.
- */
+
 static int
-npyiter_convert_ops(PyObject *op_in, PyObject *op_flags_in,
-                    PyArrayObject **op, npy_uint32 *op_flags,
-                    int *nop_out)
+npyiter_prepare_ops(PyObject *op_in, PyObject **out_owner, PyObject ***out_objs)
```

Comment:
I split this effectively, because I need the `nops`.  One helper to fetch an array of operands (with `PySequence_Fast_ITEMS`) and then the original function to conver them to arrays.

(I didn't change the fact that functions with "convert" in their name return 1 for success...)